JavaFXUserInterface/src/main/resources/SystemTypeEdit.fxml

```diff
@@ -53,69 +70,162 @@
+                                               prefWidth="350.0"/>
+                                    <Text fill="#0c072e" layoutX="82.0" layoutY="119.0" strokeType="OUTSIDE"
+                                          strokeWidth="0.0" text="Threshold: OK">
+                                        <font>
+                                            <Font name="Segoe UI Bold" size="14.0"/>
+                                        </font>
+                                    </Text>
+                                    <TextField fx:id="thresholdAdvisory" layoutX="220.0" layoutY="141.0"
+                                               prefHeight="25.0" prefWidth="350.0"/>
+                                    <Text fill="#0c072e" layoutX="82.0" layoutY="159.0" strokeType="OUTSIDE"
+                                          strokeWidth="0.0" text="Threshold: Advisory">
+                                        <font>
+                                            <Font name="Segoe UI Bold" size="14.0"/>
+                                        </font>
+                                    </Text>
+                                    <TextField fx:id="thresholdCaution" layoutX="220.0" layoutY="181.0"
+                                               prefHeight="25.0" prefWidth="350.0"/>
+                                    <Text fill="#0c072e" layoutX="82.0" layoutY="199.0" strokeType="OUTSIDE"
+                                          strokeWidth="0.0" text="Threshold: Caution">
+                                        <font>
+                                            <Font name="Segoe UI Bold" size="14.0"/>
+                                        </font>
+                                    </Text>
+                                    <TextField fx:id="thresholdWarning" layoutX="220.0" layoutY="221.0"
+                                               prefHeight="25.0" prefWidth="350.0"/>
+                                    <Text fill="#0c072e" layoutX="82.0" layoutY="239.0" strokeType="OUTSIDE"
+                                          strokeWidth="0.0" text="Threshold: Warning">
+                                        <font>
+                                            <Font name="Segoe UI Bold" size="14.0"/>
+                                        </font>
+                                    </Text>
+                                    <TextField fx:id="thresholdFailed" layoutX="220.0" layoutY="261.0" prefHeight="25.0"
+                                               prefWidth="350.0"/>
+                                    <Text fill="#0c072e" layoutX="82.0" layoutY="279.0" strokeType="OUTSIDE"
+                                          strokeWidth="0.0" text="Threshold: Failed">
+                                        <font>
+                                            <Font name="Segoe UI Bold" size="14.0"/>
+                                        </font>
+                                    </Text>
```

Comment:
This may have the dynamic issue as well. Might be misremembering what the PO said.

Reply:
Paul told me to put 5 thresholds for now with the names OK, Advisory, Caution, Warning and Failed.

Reply:
there might have been a miss-understanding, I did not say to hardcode the javafx elements with the 5 thresholds given by the po

Reply:
After further discussion, kept the hardcoded thresholds and removed the model field.